src/data/faq.json

```diff
@@ -801,7 +801,7 @@
                         "anchor": "numbers_RKI",
                         "active": true,
                         "textblock": [
-                            "The Robert Koch Institute provides these and other current numbers <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Kennzahlen.pdf?__blob=publicationFile' target='_blank' rel='noopener noreferrer'>on their website</a> (in German)."
+                            "The Robert Koch Institute provides these and other current numbers <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Archiv_Kennzahlen/WarnApp_KennzahlenTab.html;jsessionid=041B1E316F2E11A2FB9036CD0E457343.internet082?nn=13490888' target='_blank' rel='noopener noreferrer'>on their website</a> (in German)."

```

Comment:
`jsessionid=041B1E316F2E11A2FB9036CD0E457343.internet082?nn=13490888`

Session ID and nonce do not belong into this link

Reply:
Thanks, only `https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Archiv_Kennzahlen/WarnApp_KennzahlenTab.html`